stores/postDataAtom.ts

```diff
@@ -1,6 +1,17 @@
 import { atom } from "jotai";
 
-export const postTitleAtom = atom<string>("");
-export const postPublicAtom = atom<boolean>(false);
-export const postContentAtom = atom<string>("");
-export const postLastUpdatedAtom = atom<number | null>(null);
+type selectedNoteType = {
```

Comment:
Naming scheme is not correct. Types should always be in Pascal case and there's no need to mention that it's a type.

```suggestion
type SelectedNote = {
```

Reply:
@coderabbit Change the name of selectedNoteType to SelectedNote.

Reply:
@coderabbitai Change the name of selectedNoteType to SelectedNote.